tests/functional/api/BackbeatServer.js

```diff
@@ -35,16 +35,21 @@ function getUrl(options, path) {
     return `http://${options.host}:${options.port}${path}`;
 }
 
-function setKey(redisClient, keys, cb) {
-    const cmds = keys.map(key => {
-        const value = 'arn:aws:iam::604563867484:test-role';
-        return ['set', `${TEST_REDIS_KEY_FAILED_CRR}:${key}`, value];
-    });
+function addMembers(redisClient, site, members, cb) {
+    const statsClient = new StatsModel(redisClient);
+    const epoch = Date.now();
+    const twelveHouseAgo = epoch - (60 * 60 * 1000) * 12;
+    let score = statsClient.normalizeTimestampByHour(new Date(twelveHouseAgo));
```

Comment:
`_normalizeTimestampByHour`

Reply:
Oh, yes. I'm going to open a PR that changes this to a public method. Will follow with one.